pkg/agent/route/route_linux.go

```diff
@@ -431,6 +455,24 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet string, snatMa
 		"-j", iptables.MasqueradeTarget, "--random-fully",
 	}...)
 
+	if c.proxyFull {
```

Comment:
Question - why we do not put this in InitService(), or why not put InitService() code here? I think the former way is better to avoid passing/checking proxyFull in route and other code.

Reply:
If we put it to `InitService`, I think we need to write a lot of duplicated code about iptables.

Reply:
Ok, I got the point. Do not have a better idea.

Reply:
Could you add some comments to explain what rules are restored here when AntreaProxyFull is enabled?

Reply:
Got it.